components/form/index.js

```diff
@@ -1,2 +1,4 @@
+export * from './Checkbox';
 export { default as Button } from './Button';
 export { default as Input } from './Input';
+// export { default as Checkbox } from './Checkbox';
```

Comment:
If a code is not needed, it is better to remove it instead of commenting out. Also, I still don't think we need the `index.js` file declaration. Seems like extra work to me 🤷🏽‍♂️